src/app/router/connectorRoutes.tsx

```diff
@@ -21,95 +21,98 @@ import ConnectRaspiBlitz from "@screens/connectors/ConnectRaspiBlitz";
 import ConnectUmbrel from "@screens/connectors/ConnectUmbrel";
 import ConnectStart9 from "@screens/connectors/ConnectStart9";
 import ConnectMyNode from "@screens/connectors/ConnectMyNode";
+import i18n from "~/i18n/i18nConfig";
 
 const galoyPaths: { [key: string]: keyof typeof galoyUrls } = {
   bitcoinBeach: "galoy-bitcoin-beach",
   bitcoinJungle: "galoy-bitcoin-jungle",
 };
 
+const namespaceI18n = { ns: "welcome" };
```

Comment:
after looking at this again and seeing `i18n.t("choose_connector.lndhub", namespaceI18n)` in the code I thought maybe we should name this namespace to indicate which one it actually is? 

`i18n.t("choose_connector.lndhub", welcomeI18nNamespace)` ?

Reply:
Good point, it may even be useful at some point to just have the namespace variables in a dedicated file somewhere and import them as needed instead of redefining them in every file they're used.

Reply:
oh true, this could help indeed.